### PR TITLE
New command, global refactor, code style...

### PR DESCRIPTION
**Pull request by wh0am1-dev**
Hi ! 😬 

## TL;DR

- `add`: `list` synonym, `ls`
- `add`: `remove` command
- `change`: default callback is now `status`
- `fix`: remove shebangs from non-entry-points
- `refactor`: add `.editorconfig` settings
- `refactor`: code style complying with [PEP8](https://www.python.org/dev/peps/pep-0008/)
- `refactor`: add `flake8` and `setup.cfg`

## Changes

This is a huge PR -- I'd recommend viewing the commit diffs sequentially, the whole PR diff is a bit messy.

Let me explain all the changes:

- A new `list` command synonym: `ls`.
- The default command callback has been changed to `status` by default, instead of the help string. Checking the status for chronologicon is a pretty common task, as there's no other feedback to know if you are in the middle of a log or not. And it felt really natural to me checking the status just by invoking `chron`. The help string is still accessible by using any of the help commands.
- There's a new `remove` command. It takes a `logID` as a parameter and removes it from the logs. Now that I'm writing this, I've realised that I haven't provided any confirmation mechanism, though... so here's a to-do!
- I've removed the shebangs -- `#!/usr/bin/env python3` -- from all package/module files. It is supposed to work as a hint for the shell executing the script to know which interpreter to use for it. But the files inside the `chronologicon` package, in contrast with `bin/chron`, are imported and used from other scripts and not executed directly.
- I saw a couple of styling inconsistencies across the code because of the tabs. I have Sublime Text configured to render tabs as 2 chars wide, looking at files from my shell would render those as 8 chars wide, so it was like: BOOM, messy code. I've normalised tabs to be spaces. Also, I've added a `.editorconfig` -- see: [editorconfig.org](https://editorconfig.org/) -- file that tells the editor how to handle code styling. Feel free to edit these settings to your liking. Many editors / IDEs are compatible with this config file right away and many have plugins, and is a good way of avoiding potential contributors mixing tabs with spaces.
- Following the code cleanup, I've also fixed some code standards to comply with [PEP8](https://www.python.org/dev/peps/pep-0008/), which is a set of coding conventions from the standard library in the main Python distribution. With this, I've also added a `setup.cfg` with a barebones configuration for [flake8](http://flake8.pycqa.org/en/latest/). `flake8` is a linter / CLI tool to check these standards across every file in a project -- which I've added to a `requirements.txt` file for easy installation with `pip`. Again, feel free to edit this to enable/disable checking for these conventions to your liking.
- I've also modified the `.gitignore`, so it now ignores `__pycache__/` and all of the chronologicon settings / data files, to make it easier to debug / run the scripts right from the repo directory without any worries.

And... that's all ! 🎉 

PS: I'm a [joshavanier/log](https://github.com/joshavanier/log) user, and I've instantly fallen in love with this tiny tool. There are a couple of benefits, like not needing a fully fledged electron environment running to be able to use it, or seamlessly starting and ending logs from my dev scripts. So thanks for making this!

### CODE REVIEW

**Comment by wh0am1-dev** [comment]
Here's an graphical example of the [whitespace problem](https://github.com/rutherfordcraze/chronologicon/pull/3/commits/534152df5414f302bd2d6cc046742b7d47cab590#diff-658e41b32b02600ca8b3deccfd22f846)

**Comment by rutherfordcraze** [comment]
Whoah! Thanks so much for this, I'm honestly stunned at how much you've improved. I'll take a look at your changes in detail and try and incorporate as many as I can — `ls` and `remove` were both on my list of features to add, but I hadn't got around to it yet.

The dumb tab-indents are 100% my bad. I'm new to Sublime so my `.editorconfig` wasn't the same as my previous IDE.

Thanks again!

**Review by rutherfordcraze — APPROVED** [review]

**Comment by rutherfordcraze** [comment]
I don't really understand git enough to merge this PR natively, but I've made as many of the changes as I can locally.

**Comment by wh0am1-dev** [comment]
I don't want to sound like that was my objective or something, but I'm out of the contributors list... 😢 
On the other hand, I could have split this PR into several ones, my bad.

**Comment by wh0am1-dev** [comment]
Anyway, no worries, I'll probably contribute again if I keep on using chron ! 😊
